### PR TITLE
Agent docker gpu test

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Modules
- [ ] UI
- [ ] Controller
- [x] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/agent/src/main/resources/application.yaml [hunk 1]
```diff
@@ -4,5 +4,6 @@ sw:
   agent:
     task:
       infoPath: ${SW_TASK_INFO_PATH:/opt/starwhale/tasks/}
-      archivedInfoPath: ${SW_TASK_INFO_PATH:/opt/starwhale/archived-tasks/}
+      archivedInfoPath: ${SW_TASK_ARCHIVED_INFO_PATH:/opt/starwhale/archivedTasks/}
```
[the remark this answers is not in the document]
Comment (goldenxinxing) [reply server/agent/src/main/resources/application.yaml:7]: this was ide auto format,but after complied, the class will only import that need method

File: server/agent/src/main/java/ai/starwhale/mlops/agent/node/SourcePool.java [hunk 2]
```diff
@@ -14,18 +14,14 @@
...
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.concurrent.CopyOnWriteArraySet;
-import java.util.stream.Collectors;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (goldenxinxing) [reply server/agent/src/main/java/ai/starwhale/mlops/agent/node/SourcePool.java:21]: this was ide auto format,but after complied, the class will only import that need method

File: server/agent/src/main/resources/application.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (goldenxinxing) [reply server/agent/src/main/resources/application.yaml]: done

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**